### ISSUE TITLE
Payment Details section on invoice

### ISSUE THREAD

**Issue by stinethebean3** [crater-invoice/crater#1001]
**Is your feature request related to a problem? Please describe.**
Customers often have to follow up and ask for payment options or they lost where it was mentioned in previous communication.

**Describe the solution you'd like**
I've been adding this information into the Notes section, but it would be nice if payment details had it's own section and stored the account/routing numbers, BTC address, an address to mail a check, etc so it wasn't forgotten.
To go a bit further, some clients may not need all options, so being able to select which payment details for each customer, and that gets applied to recurring invoices.

**Comment by ThMountainMan** [comment]
This would be an great feature. Here in Germany it is mandetory to add all this information on the incovices. ( Tax Number / account Number .... ) So it would be great if this would be part of crater to have this at the footer of the invoice instead of manually having to edit the invoice templates with fixed variables.

**Comment by colognation** [comment]
Yes that would be really great. Maybe that could also be implemented in form of a "PDF-Footer". That could be displayed like a "general" note that can be defined in settings. It would also be helpful if the company bank details could have own fields in the settings section next do the company address.
